### DISCUSSION TITLE
Track color on hover

### DISCUSSION THREAD

**Discussion by pingel20** [jvde-github/AIS-catcher#340] [General]
I have 'Show Track on Hover' enabled and have also set a distinctive 'Track Color'. This hover feature is nice, but I would expect the track color to be the user set value.
On hovering, the track color is now the same as the 'Hover Color'.
Is this intended behaviour?

(Also the 'Track Weight' does not seem to be taken into account when hovering a ship.)

**Comment by jvde-github** [comment]
The track is colored in Hover color if result of hovering to make it stand out. So it is by design. I could change it but like that the tracks are in hover color when hovering over a ship, especially if there are a lot of tracks on the map. But always open for better options :-)

Track Weight comment, cannot replicate. What browser and version?

**Comment by pingel20** [comment]
Default Station range color and Hover color are (nearly) the same. So when I have Station range enabled the outline and the ship track sometimes overlapped. No problem anymore, I set the Station range to another color.

Track weight: set to minimum, works fine. But the track line width when hovering is wider. Minor issue. 
Latest 0.61 version with FF.

**Reply by jvde-github** [reply]
That is by design as well :-) if the ship is hovered or selected, the width of the track is 2 units wider. I guess, it might be a bit too optimistic to assume hovering color and increasing the width. Shall I remove this width addition? I am not particularly opinionated on this.

**Comment by pingel20** [comment]
No, leave it as it is. It just struck me.
The extra line width looks nice when you zoom in.

Thanks.
